trieste/acquisition/function.py

```diff
@@ -777,11 +777,16 @@ def __init__(
         search_space: SearchSpace,
         num_samples: int = 500,
         penalizer: Callable[..., PenalizationFunction] = None,
+        base_acquisition_function_builder: AcquisitionFunctionBuilder = None,
```

Comment:
Given that you only have one model, don't you want a (optional) SingleModelAcquisitionFunctionBuilder?

```suggestion
        base_acquisition_function_builder: Optional[SingleModelAcquisitionFunctionBuilder] = None,
```